src/ground_truth_nsrts.py

```diff
@@ -1002,13 +1007,15 @@ def moveto_sampler(state: State, goal: Set[GroundAtom],
         option = MoveToObj
         preconditions = {
             LiftedAtom(GripperOpen, [robot]),
+            LiftedAtom(OnTable, [targetobj]),
```

Comment:
adding this precond shouldn't affect anything, but it helps with clarity -- we only want to allow MoveToObj on objects that are on the table.